source/blog/2016-10-04-ovirt-provider-ovn.md

```diff
@@ -0,0 +1,119 @@
+
+The driver allows vdsm, libvirt, and OVN to interact whenever a NIC is pluged in such a way that the VM NIC is added to an appropriate OVN Logical Switch and the appropriate OVN overlays on all the hosts in the oVirt environment.
+
+The [oVirt OVN driver rpm](TODO: attach the RPM as late as possible to get the newest ones) is now available. The latest version can always be downloaded and built from the repository (described later in this article). Once the rpm is downloaded, it can be installed in the following way:
+
+    `dnf install ovn-provider-driver-0-1.noarch.rpm`
+
+OVN requires VDSM and OVN (version 2.6 or later) to be installed on the host. The following OVN packages must be installed:
+
+* `openvswitch`
+* `openvswitch-ovn-common`
+* `openvswitch-ovn-host`
+* `python-openvswitch`
+
+These are available from the [OVS website](http://openvswitch.org/download/) or built using the code downloaded from the OVS repo (described below).
+
+After installing the driver and OVN, the OVN-controller must be configured. This can be done either using the vdsmTool:
+
+    `vdsmTool ovn-config <OVN central IP> <local IP used for OVN tunneling>`
+
+or by using the OVN command-line interface directly. For more information about OVN-controller setup, please check the
+[OVS ducumentation](http://openvswitch.org/support/dist-docs/) 
+
+## oVirt OVN Provider
+
+The oVirt OVN provider is a proxy that the oVirt engine uses to interact with OVN. It is delivered as an rpm that is to be installed on the host where OVN central is installed.
+
+The [oVirt OVN provider RPM](attach the RPM as late as possible to get the newest ones) is also available now. The latest version can always be downloaded and built from the repository as well. Once the rpm is downloaded, it can be installed with this command:
+
+    `yum install ovn-provider-0-1.noarch.rpm`
+
+OVN requires OVN to be installed on the host (version 2.6 or later). The following OVN packages must be installed:
+
+* `openvswitch`
+* `openvswitch-ovn-common`
+* `openvswitch-ovn-central`
+* `python-openvswitch`
+
+These are also available from the [OVS website](http://openvswitch.org/download/) or built using the code downloaded from the OVS repo (described below).
+
+After installation, the provider can be started as follows:
+
+	`systemctl start ovirt-provider-ovn`
+
+The provider can then be added to oVirt as an external network provider. In order to add a new provider, go to the External Providers section in the oVirt UI and click the Add button.
+
+![adding a new provider](uarwo40-edit-cluster.png)
+
+When the provider is successfully added, existing OVN networks can be imported to oVirt. If the provider was not added as a read-only provider, new OVN networks can be defined using oVirt by adding a network and specifying it to be added on an external provider.
+
+A VM NIC can be added to OVN networks by simply choosing an external network during NIC provisioning.
```

Comment:
adding section:

OVN based networking brings many advantages to oVirt:
* More granular security - it brings complete network isolation without the need for defining VLANs. 
* Less resources - since the OVN networks are logical overlays, many of them can be defined on the same underlying physical infrastructure. As oposed to traditional oVirt networking, where each network required a separate host NIC, all the OVN networks can be acommodated on a single NIC. 
* Easier management - the management of the networking infrastructure should also become much easier, as instead of managing the network isolation on each of the networking components, it will be automatically taken care of by OVN.